TA-puppet-alert-actions/bin/ta_puppet_alert_actions/modalert_puppet_run_plan_helper.py

```diff
@@ -0,0 +1,242 @@
+
+# encoding = utf-8
+
+import json
+from puppet_plan_action import run_puppet_plan
+
+# Modular Alert Helper Functions
+#
+# override(setting_name, helper)
+# Given a setting, check to see if the alert is configured with an override value.
+def override(setting_name, helper):
+    alert_setting = helper.get_param(setting_name)
+    global_setting = helper.get_global_setting(setting_name)
+
+    if alert_setting is not None and alert_setting is not '':
+        final_value = alert_setting
+        helper.log_debug("Alert value present for '{}' it is '{}'".format(setting_name, final_value))
+    elif global_setting is not None and global_setting is not '':
+        final_value = global_setting
+        helper.log_debug("Alert value NOT present for '{}', using Global value '{}'".format(setting_name,final_value))
+    else:
+        helper.log_debug("There is no value, None returned")
+        final_value = None
+    
+    return final_value
+#
+# notnone(default_value, possible_none, helper)
+# Given a default_value and possible_value, check to make sure we are only setting values we know aren't 'None'.
+def notnone(default_value, possible_none, helper):
+    if possible_none is not None and possible_none is not '':
```

Comment:
Would `if possible_none:` work the same here?

I believe in python empty strings are falsy, as are empty sets:
```
 Here are most of the built-in objects considered false:

    constants defined to be false: None and False.

    zero of any numeric type: 0, 0.0, 0j, Decimal(0), Fraction(0, 1)

    empty sequences and collections: '', (), [], {}, set(), range(0)
```
https://docs.python.org/3/library/stdtypes.html

Reply:
Did some testing and confirmed `if possible_none:` still works as expected. 🙌

This also works as expected within the `override()` function. I'll get them updated in this file and will open a MAINT PR closer to release to clean up the few other files. 👍